src/ecosystems/maven.rs

```diff
@@ -120,14 +120,8 @@ pub struct MailingList {
     pub unsubscribe: Option<String>,
     pub post: Option<String>,
     pub archive: Option<String>,
-    #[serde(rename = "otherArchives", default)]
-    pub other_archives: Vec<OtherArchive>,
-}
-
-#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq)]
-pub struct OtherArchive {
     #[serde(rename = "otherArchive", default)]
```

Comment:
I think this should be `otherArchives` (plural) based on the [example pom.xml](https://github.com/itext/itext7/blob/2237853e4d5c4d5af7c9c5a7e28995a13cfb501e/pom.xml)

Reply:
It's a single element in the vec though. It's this element: https://github.com/itext/itext7/blob/2237853e4d5c4d5af7c9c5a7e28995a13cfb501e/pom.xml#L36

So in theory, `otherArchive` should be correct as you can see that the `otherArchives` plural is used for the parent struct.

Reply:
Definitely works now, tested it with a small standalone application:

```rs
use std::fs;

use phylum_types::ecosystems::maven;

fn main() {
    let pom = fs::read_to_string("/tmp/pom.xml").unwrap();
    let project: maven::Project = serde_xml_rs::from_str(&pom).unwrap();
    println!("Archives: {:?}", project.mailing_lists);
}
```